index.html

```diff
@@ -119,23 +126,35 @@
 <div class="container margin-top">
   <div class="row">
     <div class="col-lg-6 col-md-6 margin-bottom">
-      <img class="library-icons" src="./img/icons-library-1.png" alt="scala scalatra slick jetty">
+      <img class="library-icons" src="./img/icons-library-1.png" alt="scala scalatra slick jetty" usemap="#image-map1">
+        <map name="image-map1">
+            <area target="_blank" alt="Scala" title="Scala" href="http://www.scala-lang.org/" coords="168,6,5,57" shape="rect">
+            <area target="_blank" alt="Scalatra" title="Scalatra" href="http://scalatra.org" coords="180,2,265,61" shape="rect">
+            <area target="_blank" alt="Slick" title="Slick" href="http://slick.lightbend.com/" coords="277,3,430,59" shape="rect">
+            <area target="_blank" alt="Jetty" title="Jetty" href="http://www.eclipse.org/jetty/" coords="444,3,596,56" shape="rect">
+        </map>
     </div>
     <div class="col-lg-6 col-md-6 margin-bottom">
-      <img class="library-icons" src="./img/icons-library-2.png" alt="jGit H2 ApacheMINA IntelliJIDEA">
+      <img class="library-icons" src="./img/icons-library-2.png" alt="jGit H2 ApacheMINA IntelliJIDEA" usemap="#image-map2">
+        <map name="image-map2">
+            <area target="_blank" alt="JGit" title="JGit" href="http://www.eclipse.org/jgit/" coords="1,0,76,56" shape="rect">
+            <area target="_blank" alt="H2 Database" title="H2 Database" href="http://h2database.com/" coords="92,1,198,56" shape="rect">
+            <area target="_blank" alt="Apache MINA" title="Apache MINA" href="https://mina.apache.org" coords="211,0,331,61" shape="rect">
+            <area target="_blank" alt="IntelliJ IDEA" title="IntelliJ IDEA" href="http://www.jetbrains.com/idea/" coords="342,1,599,61" shape="rect">
+        </map>
```

Comment:
I don't have original images. We can get them by collecting from web or separating current images...